lightning/src/util/events.rs

```diff
@@ -112,8 +112,12 @@ pub enum Event {
 		/// payment is to pay an invoice or to send a spontaneous payment.
 		purpose: PaymentPurpose,
 	},
-	/// Indicates an outbound payment we made succeeded (ie it made it all the way to its target
+	/// Indicates an outbound payment we made succeeded (i.e. it made it all the way to its target
 	/// and we got back the payment preimage for it).
+	///
+	/// Note for MPP payments: we generate this event as soon as one of the payment paths succeeds. In
+	/// the case of a buggy payee, this may be followed by a PaymentFailed or MPPFragmentFailed event
```

Comment:
nit: `MPPFragmentFailed` mentioned before added

Reply:
I always mess up these types of rebases, gonna let this one slide if that's OK :sweat_smile: